source/agora/test/Flash.d

```diff
@@ -418,8 +418,9 @@ public class FlashNodeFactory (FlashListenerType = FlashListener)
     public RemoteAPI!TestFlashListenerAPI createFlashListener (
         Listener : TestFlashListenerAPI)(string address)
     {
+        const string agora_address = format("Validator #%s (%s)", 0, genesis_validator_keys[0].address);
         RemoteAPI!TestFlashListenerAPI api
-            = RemoteAPI!TestFlashListenerAPI.spawn!Listener(5.seconds);
+            = RemoteAPI!TestFlashListenerAPI.spawn!Listener(this.agora_registry, agora_address, 5.seconds);
```

Comment:
Any hope we can merge this custom impl into `Base.d` ? :D

Reply:
`FlashNodeFactory` as a whole?